client-libraries/Bash/pantry-client.bash

```diff
@@ -0,0 +1,50 @@
+# Library of bash-functions to access https://getpantry.cloud/ API
+# Just remember to put your Pantry id below.
+# Author: Somajit Dey <dey.somajit@gmail.com> 2021
+
+export PANTRY_ID="cc9c05f7-ce68-4e90-9dfa-235f2515504e"
```

Comment:
Replace your personal PantryID with "Your_Pantry_ID_here"

Reply:
Oops! Took a mental note to do that after completing the test and then forgot ;-)

In the latest commit I have changed things a little. Now PANTRY_ID needs to be passed as parameter while sourcing the script. If the user forgets, there would be warnings. Hope it's more user-friendly now.

Cheers.